test/functional/test_site/index.md

```diff
@@ -64,7 +65,52 @@ head: myCustomHead.md, myCustomHead2.md
 # Heading with hidden keyword
 <span class="keyword d-none">invisible keyword</span>
 
-<include src="testTagDivs.md" />
```

Comment:
The tags in the test looks ok. But is there any reason we moved the tags to here? The `index.md` is getting really crowded. 😅 Putting it separate would make it neater.

Reply:
Sure, moved it to separate file. Also marked the divs that should be hidden so it's easier to verify